operator/src/main/java/oracle/kubernetes/operator/DomainProcessorImpl.java

```diff
@@ -475,91 +479,107 @@ private void logThrowable(Throwable throwable) {
   }
 
   /**
-   * Begin activity to align domain status with domain resource.
-   * @param info domain presence info
-   * @param explicitRecheck if explicit recheck
+   * Begin activity to align the cached domain status with the value reaa from Kubernetes.
```

Comment:
reaa -> real

Reply:
darn it. I fixed that in the last PR in a separate commit and missed it again.